src/update.js

```diff
@@ -69,15 +70,21 @@ export default function update(data, path, operation, opts={}) {
         // Purity check
         if (opts.pure && p[s] === value)
           return {node: p[s]};
-
-        p[s] = opts.persistent ? shallowClone(value) : value;
+        
+        if(opts.persistent) {
+          p[s] = shallowClone(value);
+        } else if(value instanceof MonkeyDefinition) {
+          Object.defineProperty(p, s, {value: value, enumerable: true, configurable: true});
```

Comment:
I remember having to set configurable: true here to get replacing monkeys at runtime to work, but now the tests are green when configurable is false. I don't consider myself an expert on js, so this is confusing me somewhat... Do you have any insights?